### ISSUE TITLE
Are you able to inject with common Mono injection methods?

### ISSUE THREAD

**Issue by M4cs** [UserR00T/UniversalUnityHooks#9]
Say I wanted to make a library for use with something like MonoInjector or SharpMonoInjector. Can I include my UUH code inside the same project as my main injected library? Like if I were to just include the game's DLL and Assembly files as well as the UUH ddl as references, would it still work when Injected into the game when the game has already started?

**Comment by UserR00T** [comment]
Hi M4cs,

UniversalUnityHooks works by injecting a call assembly instruction into the MSIL code. This process is done when the UUH exe gets ran, which should happen before booting up your injected dll/game. For that reason, dynamic injection at runtime is not possible, but using something like you mentioned, SharpMonoInjector _will_ work because UUH will already have injected it's code at that point. 

Tl;dr; UUH cannot inject at runtime but will work fine with along with the injectors that can.

I hope this answered you question and if it did feel free to close the issue. If you got any more questions let me know and I'll try to respond to them as soon as possible.

**Comment by M4cs** [comment]
Thanks for the response! That makes a lot more sense to me.

So to follow up, the flow would be:

Inject UUH Code into Assembly-CSharp.dll > Run Game > Inject with a Mono Injector?

Would that mean that UUH could set off Anti-Cheat if there are DLL checksums in the game?
